QuickBytes/src/main/java/com/quickbytes/controller/RequestComboController.java

```diff
@@ -0,0 +1,142 @@
+package com.quickbytes.controller;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.hibernate.cache.spi.support.AbstractReadWriteAccess.Item;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.quickbytes.model.Combo;
+import com.quickbytes.model.Request;
+import com.quickbytes.model.RequestCombo;
+import com.quickbytes.repository.RequestRepository;
+import com.quickbytes.repository.ComboRepository;
+import com.quickbytes.repository.RequestComboRepository;
+
+@RestController
+public class RequestComboController {
+	
+	@Autowired
+	RequestComboRepository requestComboRepository;
+	
+	@Autowired
+	RequestRepository requestRepository;
+	
+	@Autowired
+	ComboRepository comboRepository;
+	
```

Comment:
Correct me if I'm wrong but I believe that "base urls" should have a post method as the foundation for the rest of the functionality.

i.e.
@PostMapping("/requestcombo")
public void postRequestCombo(@......){
saving passed RequestCombo within RequestComboRepository
}

Reply:
Done but I'm curious as to why?